### PR TITLE
Add missing space causing bash issue

### PR DESCRIPTION
**Pull request by lekaf974**
**Notes for Reviewers**

Fix nightly cleanup issue https://github.com/meshery/meshery/actions/runs/8515410179/job/23322835155#step:2:75

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Review by cpepper96 — APPROVED** [review]
LGTM

File: install/playground/playground-cleanup.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #Script is placed in /root directory in VM.
 for ns in $(kubectl get ns -o jsonpath="{.items[*].metadata.name}");
 do
-    if [[ "$ns" != "kube-system" && "$ns" != "default" && "$ns" != "monitoring" && "$ns" != "kube-flannel" && "$ns" != "kube-node-lease" && "$ns" != "kube-public" && "$ns" != "meshery" && "$ns" != "metallb-system" && "$ns" != "projectcontour" && "$ns" != "ingress-nginx" && "$ns" != "monitoring" && "$ns" != "layer5-cloud" && "$ns" != "postgres"]];then
+    if [[ "$ns" != "kube-system" && "$ns" != "default" && "$ns" != "monitoring" && "$ns" != "kube-flannel" && "$ns" != "kube-node-lease" && "$ns" != "kube-public" && "$ns" != "meshery" && "$ns" != "metallb-system" && "$ns" != "projectcontour" && "$ns" != "ingress-nginx" && "$ns" != "monitoring" && "$ns" != "layer5-cloud" && "$ns" != "postgres" ]]; then
```
Comment (cpepper96) [inline install/playground/playground-cleanup.sh:6]: Actually I just realized we check if the namespace is "monitoring" twice in this if statement. I would remove the duplicate `"$ns" != "monitoring"` and then it LGTM @lekaf974

Comment (lekaf974) [reply install/playground/playground-cleanup.sh:6]: @cpepper96 I'll update it soon

Also, by using an array we could make the check more readable and will avoid duplication.
What do you think about it ?

**Comment by vishalvivekm** [comment]
@lekaf974 Let's discuss this in Meshery dev call today. I'm adding this as an agenda item to the meeting[ minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#).

File: install/playground/playground-cleanup.sh (hunk 1 above)
Comment (lekaf974) [reply install/playground/playground-cleanup.sh:6]: an example/test of my suggestion
image

**Review by leecalcote — APPROVED** [review]
Great feedback from, @cpepper96 and @sangramrath.

LGTM

**Comment by sangramrath** [comment]
LGTM as well, the updated script with an array approach as proposed by @lekaf974.  
May I suggest "DONOTDELETE_NAMESPACES" instead of "valid_namespaces"? :)

**Review by MUzairS15 — APPROVED** [review]

**Pull request merged**